src/chatbot/run_chatbot.py

```diff
@@ -25,6 +31,12 @@
 B: Najbardziej lubię filmy akcji.
 ###"""
 NUM_CONVERSATION_SAMPLES = len(CONVERSATION_SAMPLES.split('###')) - 1
+# Downloaded from https://github.com/sdadas/polish-nlp-resources
+WORD_TO_VEC_PATH = "/home/maria/Downloads/word2vec/word2vec_100_3_polish.bin"
+DEBUG = True
```

Comment:
Don't we want DEBUG=False in main?

Reply:
Also why don't we make this path relative?